lib/index.js

```diff
@@ -39,7 +39,13 @@ internals.implementation = function (server, options) {
       // otherwise use the same key (String) to validate all JWTs
       var decoded;
       try {
-        decoded = JWT.decode(token); // decode is non-io and fast enough to not have to be async
+		if (options.completeToken === true) { 
```

Comment:
any reason we cannot simply write:
```js
decoded = JWT.decode(token, { complete: options.complete || true });
```
Is there a *massive* performance penalty for doing a "*complete*" decode?